content/blog/2024-12-21-how-do-i-...-in-dt-5.0/index.md

```diff
@@ -0,0 +1,508 @@
+---
+title: "How do I ... in darktable 5.0?"
+author: "Ralf Brown"
+slug: howto-in-5.0
+date: 2024-12-20
+lede: falls.jpg
+lede_author: "Ralf Brown"
+tags:
+  - howto
+---
+
+Users new to darktable who have experience with other raw developers
+may have trouble figuring out how to do various tasks as a result of
+different names or workflows in darktable compared to their previous
+software.  The following entries are a quick reference to help you
+accomplish what you want to do.
+
+Have a tip not covered here?  Share it in the comments!
+
+### How do I...
+
+<details><summary><b>run darktable in "simple" or "beginner" mode</b></summary>
```

Comment:
I don't understand the <details> and <summary> markup here, why are these not just standard axe style markdown headers?

Reply:
How do I get a collapsible section in pure Markdown?  *Every* article I've found about the topic says you need to use the \<details\> HTML tag....

Reply:
Additional searching found a Hugo-specific way to implement this.  Which also seems to remove the need to change normalize.css....